bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/common/WeatherConfig.java

```diff
@@ -167,8 +186,7 @@ private void parseApiKey(String key, String value) throws ConfigurationException
         } else if (StringUtils.equalsIgnoreCase(keyId, "apikey2")) {
             pConfig.setApiKey2(value);
         } else {
-            throw new ConfigurationException("weather",
-                    "Unknown configuration key '" + key + "', please check your openhab.cfg!");
+            logger.debug("Unknown configuration key '{}'. Check openhab.cfg.", key);
```

Comment:
The reason I prefer warn over debug in this case is that a user might mistype the configuration key.  So with debug-only logging of the unrecognized key, the user will think they set the configuration correctly, but the default value is being used instead.  By making it warn-level, I think it's more likely the user will see their mistake.

All that said, it's a very minor issue, so if you're not persuaded, then leave it as is.